source/go-perun/tutorial/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _developer-tutorial:
+
+Tutorial
+========
+
+This tutorial describes how to build a simple payment channel application on top of Ethereum using `go-perun <https://github.com/hyperledger-labs/go-perun>`_.
+The tutorial targets developers familiar with Golang and Ethereum.
+The source code is available at `GitHub <https://github.com/perun-network/perun-tutorial>`_.
```

Comment:
If you're including the source code from a third-party under a different license, then the license obligations need to be fulfilled. Provide license, NOTICE file if available, ...
```suggestion
The source code snippets within this tutorial are literally included from *perun-tutorial*, (c) PolyCrypt GmbH 2021, licensed under `Apache-2.0 <https://github.com/perun-network/perun-tutorial/blob/master/LICENSE>`_. The entire source code is available at `GitHub <https://github.com/perun-network/perun-tutorial>`_.
```

Reply:
ok